src/RulesTest/Library/BoundingRectangleCompletelyObscuresContainerTest.cs

```diff
@@ -1,11 +1,12 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Core.Bases;
+using Axe.Windows.Core.Types;
 using System.Drawing;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Axe.Windows.Rules.PropertyConditions;
```

Comment:
Can we sort these using statements alphabetically while we're here?
```suggestion
using Axe.Windows.Core.Bases;
using Axe.Windows.Core.Types;
using Axe.Windows.Rules.PropertyConditions;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System.Drawing;
```

Reply:
Done.